libs/framework/src/celix_launcher.c

```diff
@@ -256,11 +267,47 @@ static void printEmbeddedBundles() {
     celix_arrayList_destroy(embeddedBundles);
 }
 
-static void combine_properties(celix_properties_t *original, const celix_properties_t *append) {
+static int celixLauncher_createBundleCache(celix_properties_t* embeddedProperties, const char* configFile) {
+    celix_framework_t* fw = NULL;
+    celix_properties_t* config = celixLauncher_createConfig(configFile, embeddedProperties);
+    celix_status_t status = framework_create(&fw, config);
```

Comment:
In `celixLauncher_printProperties`, it is clear that `env` > `configFile` > `embeddedProperties` in priority. 
However, in `framework_create`, `env` is ignored for `CELIX_FRAMEWORK_STATIC_EVENT_QUEUE_SIZE`.

Reply:
correct. nice catch. Added usage of celix_framework_getConfigProperty to remedy this.